tdrs-backend/plg/grafana/dashboards/health.json

```diff
@@ -0,0 +1,1457 @@
+{
+  "annotations": {
```

Comment:
ask alex about user experience on filters - selecting individual vs having copies of dashboard for each environment

Reply:
@jtimpe i dont mind selecting env then app. i did switch the order of these env vars since this is the typical workflow. let me know if this could cause problems. 

![Screenshot 2025-03-26 101257](https://github.com/user-attachments/assets/7cd94cf7-98c0-4566-ae01-8f6b3288f354)